src/components/base/view.js

```diff
@@ -231,3 +251,14 @@ genotet.View.prototype.createPanel = function(container) {
 genotet.View.prototype.signal = function(eventType, opt_data) {
   $(this).trigger('genotet.' + eventType, [opt_data]);
 };
+
+/**
+ * Sets the size of container.
+ * @param {number} containerWidth Width of container.
+ * @param {number} containerHeight Height of container.
+ */
+genotet.View.prototype.resize = function(containerWidth,
+                                                      containerHeight) {
```

Comment:
The indentation is a little weird here. Suggest aligning containerHeight with containerWidth.